docs/flink-getting-started.md

```diff
@@ -747,8 +724,8 @@ DataStream<org.apache.avro.generic.GenericRecord> dataStream = ...;
 
 Schema icebergSchema = table.schema();
 
-// if the Iceberg table schema contains time fields, we can't use
-// Avro schema converted from Iceberg schema via AvroSchemaUtil.
+// if the Iceberg table schema contains time fields, and can't use
+// the Avro schema converted from Iceberg schema via AvroSchemaUtil.
```

Comment:
nit: seems odd syntax. maybe `the Avro schema converted from Iceberg schema can't be used due to precision difference between how Iceberg schema (micro) and Flink {@link AvroToRowDataConverters} (milli) deal with time type`